config/job.yaml

```diff
@@ -0,0 +1,55 @@
+# Job Configuration
+
+job:
+    id: 
+    cluster_configuration:
+        vm_size: 
+        size: 
+        size_low_pri: 
+        docker_repo: 
+        custom_scripts: 
+
+    spark_configuration:
+        spark_defaults_conf: 
+        spark_env_sh: 
+        core_site_xml: 
+  
+    applications:
```

Comment:
we should specify that an application is the equivalent to a spark submit command as a comment